### DISCUSSION TITLE
Повтор текущего трека в Радио

### DISCUSSION THREAD

**Discussion by c01nd01r** [Chimildic/YaMuTools#14] [Идея] [answered]
Привет!
Было бы круто иметь кнопку повтора трека, который играет в данный момент в Радио.
Бывает, попадется интересная композиция и хочется слушать ее на репите :)
Но для этого надо открыть текущий трек в новой вкладке, включить его там, нажать кнопку повтора трека... В общем, много возни.

**Comment by Chimildic** [comment, marked as answer]
Добавил кнопку репита к плееру на сайте Музыки и Радио (версия 0.7.3). Отправил обновление на проверку в Web Store.

**Reply by c01nd01r** [reply]
Офигенно, спасибо! 🔥
